index.html

```diff
@@ -126,6 +126,17 @@ <h2>
       </p>
 
       <p>The <dfn>devicechange</dfn> event is defined in [[GETUSERMEDIA]].</p>
+
+      <p>
+        A document is called an <dfn class="export">actively screen-capturing document</dfn>
+        if that document holds a reference to a {{MediaStreamTrackState/"live"}} video track
+        derived from a call to {{MediaDevices/getDisplayMedia}}.
```

Comment:
Should there be a camera/microphone specific version or should it be a generic version (camera/microphone/display capture)?
The latter would be defined in mediacapture-main by checking if the document  [[devicesLiveMap]] has any value being set to true.

We might also want to take muted into account. If all tracks are muted=true, is the document still actively screen capturing?
enabled=false is also something we should think about.

Reply:
@kenchris

Reply:
Sorry missed this. Yes, generic version would be preferred, and I think it makes sense to consider muted